test/integration-test.py

```diff
@@ -67,26 +66,6 @@ def test_single_ocsp():
     p.send_signal(signal.SIGTERM)
     p.wait()
 
-def test_stats():
-    """Fetch Prometheus metrics from a sample of Boulder components to check
-       they are present.
-
-       This is a non-API test.
-    """
-    def expect_stat(port, stat):
-        url = "http://localhost:%d/metrics" % port
-        response = requests.get(url)
-        if not stat in response.text:
-            print(response.content)
-            raise(Exception("%s not present in %s" % (stat, url)))
-    expect_stat(8000, "\nresponse_time_count{")
-    expect_stat(8000, "\ngo_goroutines ")
-    expect_stat(8000, '\ngrpc_client_handling_seconds_count{grpc_method="NewRegistration",grpc_service="ra.RegistrationAuthority",grpc_type="unary"} ')
-
-    expect_stat(8002, '\ngrpc_server_handling_seconds_sum{grpc_method="PerformValidation",grpc_service="ra.RegistrationAuthority",grpc_type="unary"} ')
-
-    expect_stat(8001, "\ngo_goroutines ")
```

Comment:
These last two are checking stats on CA and RA, did you intend to delete these as well?

Reply:
We covered this on the call. All good from my end!

Reply:
Yep, we have a go test for wfe2 metrics [here](https://github.com/letsencrypt/boulder/blob/4c8d4758842ae1900a1bb04a8eea8314366671de/test/integration/wfe_test.go#L38) and a python test for "metrics from every other service" [here](https://github.com/letsencrypt/boulder/blob/4c8d4758842ae1900a1bb04a8eea8314366671de/test/integration-test.py#L223), so this deleted test is just duplicate functionality.